src/frontend/src/binder/expr/function.rs

```diff
@@ -839,6 +839,8 @@ impl Binder {
                 ("array_sum", raw_call(ExprType::ArraySum)),
                 ("array_position", raw_call(ExprType::ArrayPosition)),
                 ("array_positions", raw_call(ExprType::ArrayPositions)),
+                ("array_contains", raw_call(ExprType::ArrayContains)),
+                ("array_contained", raw_call(ExprType::ArrayContained)),
```

Comment:
I chose the fourth, It would be better if pg users can get the same behavior using the same sql. Keep another form of functions as other array function as clearness